cmd/byctl/ls.go

```diff
@@ -47,14 +47,10 @@ var lsCmd = &cli.Command{
 			return err
 		}
 
-		format := shortListFormat
-		if c.Bool("l") || c.String("format") == "long" {
-			format = longListFormat
-		}
-
-		isFirstSrc := true
-		for i := 0; i < c.Args().Len(); i++ {
-			conn, path, err := cfg.ParseProfileInput(c.Args().Get(i))
+		// parse args
+		var storePathMap = make(map[*operations.SingleOperator][]string)
```

Comment:
The map here looks strange. I'm not sure whether to expand all the globs first, or one by one (expand one and then execute `List`).